bootstrap/user_settings.yaml

```diff
@@ -1,6 +1,9 @@
 # PyExpLabSys settings defaults
 
 # Common
+sql_server_host: 'fys-plog01.win.dtu.dk'
```

Comment:
I guess lines like this should all be replaced by something that is meaningful, but invalid like: 'sql_server_hostname.example.com'
In a single installation this can be modified manually. For larger installations, the organisation can have their own template replace this one.

Reply:
Agreed, I have updated accordingly 👍